src/Scaler.Demo/OrderGenerator/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "CosmosDbConfig": {
-    "Connection": "<connection-string-of-monitored-container-account>",
+    "Endpoint": "https://{Cosmos Account Name}.documents.azure.com:443/",
```

Comment:
In case of Order Generator, we are not deploying it on AKS. It just used locally to generate CosmosDB records and trigger the Change Feed, hence Order Generator is using connection string method to authenticate.